src/pages/workspace/accounting/intacct/export/SageIntacctReimbursableExpensesPage.tsx

```diff
@@ -0,0 +1,126 @@
+    const {reimbursable, reimbursableExpenseReportDefaultVendor} = policy?.connections?.intacct?.config?.export ?? {};
+
+    const data: MenuListItem[] = Object.values(CONST.SAGE_INTACCT_REIMBURSABLE_EXPENSE_TYPE).map((expenseType) => ({
+        value: expenseType,
+        text: translate(`workspace.sageIntacct.reimbursableExpenses.values.${expenseType}`),
+        keyForList: expenseType,
+        isSelected: reimbursable === expenseType,
+    }));
+
+    const selectReimbursableDestination = useCallback(
+        (row: MenuListItem) => {
+            if (row.value !== reimbursable) {
+                updateSageIntacctReimbursableExpensesExportDestination(policyID, row.value);
+            }
+            if (row.value === CONST.SAGE_INTACCT_REIMBURSABLE_EXPENSE_TYPE.VENDOR_BILL) {
+                Navigation.goBack(ROUTES.POLICY_ACCOUNTING_SAGE_INTACCT_EXPORT.getRoute(policyID));
+            }
+        },
+        [reimbursable, policyID],
+    );
+
+    const defaultVendorSection = {
+        description: translate('workspace.sageIntacct.defaultVendor'),
+        action: () => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_SAGE_INTACCT_DEFAULT_VENDOR.getRoute(policyID, CONST.SAGE_INTACCT_CONFIG.REIMBURSABLE)),
+        title: reimbursableExpenseReportDefaultVendor
+            ? getDefaultVendorName(reimbursableExpenseReportDefaultVendor, intacctData?.vendors ?? [])
+            : translate('workspace.sageIntacct.notConfigured'),
+        hasError: !!config?.export?.errorFields?.reimbursableExpenseReportDefaultVendor,
+        pendingAction: config?.export?.pendingFields?.reimbursableExpenseReportDefaultVendor,
+    };
+
+    const defaultVendor = (
+        <OfflineWithFeedback
+            key={defaultVendorSection.description}
+            pendingAction={defaultVendorSection.pendingAction}
+        >
+            <MenuItemWithTopDescription
+                title={defaultVendorSection.title}
+                description={defaultVendorSection.description}
+                shouldShowRightIcon
+                onPress={defaultVendorSection.action}
+                brickRoadIndicator={defaultVendorSection.hasError ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
+            />
+        </OfflineWithFeedback>
+    );
+
+    return (
+        <ConnectionLayout
+            headerTitle="workspace.sageIntacct.reimbursableExpenses.label"
+            title="workspace.sageIntacct.reimbursableExpenses.description"
+            titleStyle={[styles.ph5, styles.pb5]}
+            onBackButtonPressRoute={ROUTES.POLICY_ACCOUNTING_SAGE_INTACCT_EXPORT.getRoute(policyID)}
+            accessVariants={[CONST.POLICY.ACCESS_VARIANTS.ADMIN, CONST.POLICY.ACCESS_VARIANTS.PAID]}
+            featureName={CONST.POLICY.MORE_FEATURES.ARE_CONNECTIONS_ENABLED}
+            displayName={SageIntacctReimbursableExpensesPage.displayName}
+            policyID={policyID}
+            connectionName={CONST.POLICY.CONNECTIONS.NAME.SAGE_INTACCT}
```

Comment:
I think we should add `shouldUseScrollView={false}` to avoid this warning, or just switch to use `SelectionScreen`.
```suggestion
            connectionName={CONST.POLICY.CONNECTIONS.NAME.SAGE_INTACCT}
            shouldUseScrollView={false}
```

https://github.com/Expensify/App/assets/16502320/0c5a937e-f4c3-4ee4-82e3-18561e6de9ed

Reply:
We shouldn't use SelectionScreen here because we have some additional content below the list (toggle and default vendor). I'll update it to set `shouldUseScrollView` to false (I already did it in my offline and error PR but forgot to add it here 🤦)

Reply:
Cool cool. So we can leave it there since you fixed it in your follow up PR. Please resolve the conflict 😄

Reply:
Conflict resolved (locally). I'm updating the `shouldUseScrollView` here (to get rid of error). I'll test it and then push changes. 
Should be there in few minutes :)